main.go

```diff
@@ -120,7 +122,25 @@ func tagFileResources(path string, dir string, filter string, tags string, tfVer
 			} else {
 				log.Print("[INFO] Resource not taggable, skipping.", resource.Labels())
 			}
+		case "locals":
+			// Checks if terratag_added_* exists.
+			// If it exists no need to append it again to Terratag file.
+			// Instead should override it.
+			attributes := resource.Body().Attributes()
+			key := tag_keys.GetTerratagAddedKey(filename)
+			for attributeKey, attribute := range attributes {
+				if attributeKey == key {
+					mergedLocals, err := convert.MergeLocals(attribute, terratag.Added)
+					if err != nil {
+						errors.PanicOnError(err, nil)
+					}
+					terratag.Added = mergedLocals
```

Comment:
If I'm not mistaken, this won't work if user has more than 1 `locals` block (not terratag-generated)

Reply:
Sorry for repeating myself. This only works on the terratag local. It will skip all other locals.

Reply:
I misunderstood. Summary [here](https://github.com/env0/terratag/pull/118/files#r854117741)